config/phpstan-baseline.neon

```diff
@@ -42,12 +42,6 @@ parameters:
 			count: 1
 			path: ../src/HtmlProcessor/CssVariableEvaluator.php
 
-		-
-			message: '#^Parameter \#2 \$callback of function preg_replace_callback expects callable\(array\<string\>\)\: string, Closure\(array\<int, string\>\)\: string given\.$#'
-			identifier: argument.type
-			count: 1
-			path: ../src/HtmlProcessor/CssVariableEvaluator.php
-
```

Comment:
Looks like the `thecodingmachine/phpstan-safe-rule` library has a solution for https://github.com/phpstan/phpstan/issues/13824 when using the `Safe` functions.